plans/example-browser-node/scripts/polyfill.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# polyfill setimmediate for winston in browser
+sed "1s/^/require('setimmediate')\n\n/" src/index.js > src/index_polyfill.js
+mv src/index_polyfill.js src/index.js
```

Comment:
Could you share some context here? Could we use the `browser` field? https://github.com/defunctzombie/package-browser-field-spec

Reply:
Not sure, might need some help there, I am certainly no expert in the NodeJS world. What would be the diference with our current approach? It's certainly not the case that I can just run winston as-is without any shimming in the browser, unless you just need to enable some config for that?!

Reply:
(I think we developed this discussion in the js-sdk already and this can be resolved, let me know if you have more questions)